### ISSUE TITLE
TrailingSemicolon warning for @include blocks

### ISSUE THREAD

**Issue by blmyr** [brigade/scss-lint#205]
Unfortunately #198 and #200 are not fixed completely. If there is no CSS selector or property in the first block there is still a warning. Take a look at this example:

``` bash
$ cat test.scss      
@include foo(bar) {
  @if $var {
    color: #f00;
  }
}

@include foo(bar) {
  background: #000;
  @if $var {
    color: #f00;
  }
}
$  scss-lint test.scss
test.scss:1 [W] TrailingSemicolon: Declaration should not have a space before the terminating semicolon
$  scss-lint -version
scss-lint 0.26.1
```
